common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -362,17 +362,6 @@ d_rustbolt = {
 	}
```

Comment:
This CoA should be a bit highter.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122671598-8639ad00-d1d8-11eb-8685-7216cb03aa53.png)

</details>